phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/installations/dto/InstallationDTO.java

```diff
@@ -61,16 +65,19 @@ private InstallationDTO(Builder builder) {
         node = builder.node;
     }
 
+    /**
+     * type of the installation
+     */
     public enum InstallationType {
-        APPS, WINE;
+        APPS, ENGINES;
 
         @Override
         public String toString() {
             switch (this) {
```

Comment:
As an alternative you could add the display name of the installation types as a parameter to the enum, then you would simply need to return this parameter.